src/integrationTest/java/org/opensearch/security/privileges/int_tests/ClusterConfig.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.privileges.int_tests;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import org.opensearch.test.framework.cluster.LocalCluster;
+
+/**
+ * This is one of the test parameter dimensions used by the *Authorization*IntTests test suites.
+ * The test suites run on different cluster configurations; the possible cluster configurations are defined here.
+ */
+public enum ClusterConfig {
+    LEGACY_PRIVILEGES_EVALUATION(
```

Comment:
To make sure I understand correctly, `legacy` here is related to how DNFOF relates to system index authorization?

Reply:
Good point. 

The naming is already in anticipation of the enhanced privilege evaluation PR where the enum looks like this:

https://github.com/opensearch-project/security/blob/b3e2cca6b2417da218152649dfa283e3fea6d06a/src/integrationTest/java/org/opensearch/security/privileges/int_tests/ClusterConfig.java#L20-L38

Thus, currently, this is not really legacy. But maybe we can keep it this way, as it is only temporary and avoids lots of renaming?

But, generally, I am also not sure about a good way to name these things. Also I am not too happy about this "next_gen" name.